contracts/peg/PegStabilityModule.sol

```diff
@@ -1,13 +1,18 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity ^0.8.4;
 
-import "./../pcv/PCVDeposit.sol";
-import "./../fei/minter/RateLimitedMinter.sol";
-import "./IPegStabilityModule.sol";
-import "./../refs/OracleRef.sol";
-import "../Constants.sol";
-import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
-import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {Decimal} from "../external/Decimal.sol";
+import {OracleRef} from "./../refs/OracleRef.sol";
+import {CoreRef} from "./../refs/CoreRef.sol";
+import {Constants} from "../Constants.sol";
+import {RateLimitedMinter} from "./../fei/minter/RateLimitedMinter.sol";
+import {IPegStabilityModule} from "./IPegStabilityModule.sol";
+import {IPCVDeposit, PCVDeposit} from "./../pcv/PCVDeposit.sol";
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
```

Comment:
Looks like there are a lot of extra imports here that aren't used

Reply:
all of those imports are needed

Reply:
try removing any of them them and see what happens